Apps/WebClient/src/ClientApp/src/views/landing.vue

```diff
@@ -85,6 +78,13 @@ export default class LandingView extends Vue {
             color: "",
             active: false,
         },
+        {
+            name: "Laboratory",
+            definition: "exclamation-triangle",
+            label: "COVID-19 Test Results (Sep 2020)",
```

Comment:
This is the COVID list item, note the lab one

Reply:
good catch!